crypto/src/test/java/io/neow3j/crypto/transaction/RawInvocationScriptTest.java

```diff
@@ -0,0 +1,95 @@
+package io.neow3j.crypto.transaction;
+
+import io.neow3j.crypto.ECKeyPair;
+import io.neow3j.crypto.Keys;
+import io.neow3j.crypto.Sign;
+import io.neow3j.crypto.Sign.SignatureData;
+import io.neow3j.io.NeoSerializableInterface;
+import io.neow3j.utils.ArrayUtils;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.util.Arrays;
+
+import static io.neow3j.constants.OpCode.PUSHBYTES64;
+import static org.junit.Assert.assertArrayEquals;
+
+public class RawInvocationScriptTest {
+
+    @Test
+    public void testFromMessageAndKeyPair() throws InvalidAlgorithmParameterException,
+            NoSuchAlgorithmException, NoSuchProviderException {
+
+        byte[] message = new byte[10];
+        Arrays.fill(message, (byte) 10);
+        ECKeyPair keyPair = Keys.createEcKeyPair();
+        RawInvocationScript invScript = RawInvocationScript.fromMessageAndKeyPair(message, keyPair);
+        SignatureData expectedSignature = Sign.signMessage(message, keyPair);
+        byte[] expectedScript = ArrayUtils.concatenate(PUSHBYTES64.getValue(), expectedSignature.getConcatenated());
```

Comment:
Well, I prefer to use a String with hexadecimal values and commenting them on the test itself -- for a better understanding.
E.g.:
```
byte[] expectedScript = Numeric.hexStringToByteArray(
    // PUSHBYTES64
    "64" +
    // signature
    "00000000000000000000000"
);
```

But it's a matter of style as well. The disadvantage is that such string can get pretty large. The advantage is a better understanding of how things are built.
Anyway, testing and making it work is more important. 👍 :-)